incent-config/Dangerfile

```diff
@@ -218,4 +218,4 @@ fail('Please only modify one program-set at a time') if multiple_program_sets?
 
 fail('Please do not add Rakefiles to config repos') if rakefile_added?
 
-fail('Please add a description for your PR') if github.pr_body.empty?
+fail('Please add a description for your PR') if github.pr_body.empty?
```

Comment:
Don't know whats going on here??

Reply:
Maybe there was or wasn't a newline at the end of the file. 🤷‍♂️